### DISCUSSION TITLE
Docker container modify cronjob yaml

### DISCUSSION THREAD

**Discussion by danfecd** [firefly-iii/firefly-iii#9020] [Firefly III] [answered]
Hi,
First Thanks James for the amazing job I love your project.
I'm trying to set the cron job docker container. Until now, following the instructions of the page https://docs.firefly-iii.org/how-to/firefly-iii/advanced/cron/   I'am able to run basic cron jobs, however I want to launch the job as if it was a day in the past. 
I tried to use this command in the yaml file --force --date=$(date "+\%Y-\%m-")10 but without success. 
I tried something like command: sh -c "echo \"* 3  10 * * wget -qO- http://app:8080/api/v1/cron/[TOKEN]\  --force --date=$(date "+\%Y-\%m-")1 " | crontab - && crond -f -L /dev/stdout"
I want to run the cron the 10th pretending to be the 1st of th month 
Do you know how can I implement it using the cron container.
Thanks for any help you can provide

**Comment by JC5** [comment]
Check out:

https://github.com/orgs/firefly-iii/discussions/8759

**Reply by danfecd** [reply]
Thanks, but I have fireflyiii installed as a docker container, the cron also as a container, I assume the forced day should be passed as a header in the url, am I wrong?
I think in this discussion https://github.com/orgs/firefly-iii/discussions/8759 the app is installed directly in the OS so the cron might be different to the one used with the containers.

**Reply by danfecd** [reply, marked as answer]
Hi 
I found the solution looking on the API documentation, and the post you sent me gave me an idea. What I did was.

1) I created the cron container with a dummy cronjob following the documentation
2) Then sh into the container (docker exec -it  [cron container] /bin/sh
3) I edit the crontab (crontab -e) and modify the end of the url, should look something like  
ht<span>tp://</span>app:8080/api/v1/cron/[TOKEN]?date=$(date +01-%m-%Y)&force=true
